Examples/test_dealer.pl

```diff
@@ -25,5 +25,8 @@
      print "Comparing against reference output from $refer\n";
      print `diff $output $refer`;
   }
+ 
+  $seed++;
```

Comment:
Ohh, this is likely why the other Refer.* files changed! When you change the rng seed, of course the output will differ slightly. but I should probably go through those files anyway and make sure the differences are indeed slight and random-looking.